test/parallel/test-util-inherits.js

```diff
@@ -75,6 +75,12 @@ assert.strictEqual(e.e(), 'e');
 assert.strictEqual(e.constructor, E);
 
 // should throw with invalid arguments
-assert.throws(function() { inherits(A, {}); }, TypeError);
-assert.throws(function() { inherits(A, null); }, TypeError);
-assert.throws(function() { inherits(null, A); }, TypeError);
+assert.throws(function() { inherits(A, {}); },
+              common.expectsError({ code: 'ERR_NO_PROTOTYPE',
+                                    type: TypeError }));
+assert.throws(function() { inherits(A, null); },
+              common.expectsError({ code: 'ERR_REQUIRED_ARG',
+                                    type: TypeError }));
+assert.throws(function() { inherits(null, A); },
+              common.expectsError({ code: 'ERR_REQUIRED_ARG',
+                                    type: TypeError }));
```

Comment:
Not for this PR, but separately we should probably provide some common guidance as to whether the tests should validate the message.